firefox-webext/background.js

```diff
@@ -5,6 +5,26 @@
 var prefix = 'https://webcompat.com/?open=1&url=';
 var screenshotData = '';
 
+chrome.contextMenus.create({
+  id: "webcompat",
+  title: "Report Website compatibility issue",
```

Comment:
This should probably be shorter since it's in a context menu (although I don't see a limit...). Maybe `Report site issue`? That's what the Fennec Nightly/Aurora menu option says is.